autobuild/mac/codeQL/autobuild_mac_1_prepare.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# autobuild_1_prepare: set up environment, install Qt & dependencies
+
+###################
+###  PROCEDURE  ###
+###################
+
+echo "Install dependencies..."
+brew install qt5
```

Comment:
Todo: use aqt and the same process as in the release process.

Reply:
There should only be one "Install Qt" (per platform).  If it's always needed, always run it (as a workflow step).  One workflow.  Some steps always run (i.e. those that prepare for what happens later).  Some steps only run under certain conditions (branch name, action, target - CodeQL only runs sometimes, publish only runs sometimes).

("target" - this doesn't just mean platform, it includes combinations like "headless server on debian x64".)